core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -42,20 +41,20 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
       });
   }
 
-  getExecutionStatus(statusCode: number): String {
+  getExecutionStatus(statusCode: number): string[] {
```

Comment:
Please add a comment that the colors match those in this [file](https://github.com/Texera/texera/blob/f991f1315bf3a2de6e9478ac0abf37610f9ad995/core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts#L348) so that if one is changed he other should also be changed.
Additionally, add a TODO comment to move those colors in a config file to change them once not in many places